common/src/main/java/org/jboss/hal/testsuite/category/Eap.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2015-2016 Red Hat, Inc, and individual contributors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.hal.testsuite.category;
+
+/** JUnit category for JBoss EAP product specific tests. */
+public interface Eap {
```

Comment:
Could you change it to EAP please? Eap just doesn't look right for me :upside_down_face: :